### PR TITLE
Update workflow to add kuberay in compatibility matrix

### PR DESCRIPTION
**Pull request by ChughShilpa**
# Issue link

Closes #<[380](https://github.com/project-codeflare/codeflare-operator/issues/380)>

# What changes have been made
Updating `Tag and build`GH action to add KubeRay line in the compatibilty matrix

# Verification steps

## Checks
- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: .github/workflows/tag-and-build.yml [hunk 1]
```diff
@@ -24,6 +24,10 @@ on:
         description: 'Published version of InstaScale'
         required: true
         default: 'v0.0.0-dev'
+      kuberay-version:
+        description: 'Published version of KubeRay'
```
Comment (astefanutti) [inline .github/workflows/tag-and-build.yml:28]: We don't publish a KubeRay version strictly speaking. So maybe we can describe it as "Tested version of KubeRay"?

Comment (ChughShilpa) [reply .github/workflows/tag-and-build.yml]: Updated the description

**Review by sutaakar — APPROVED** [review]
/lgtm

**Pull request merged**